packages/application-shell/src/components/gtm-user-tracker/gtm-user-tracker.js

```diff
@@ -21,7 +21,7 @@ class GtmUserTracker extends React.PureComponent {
     if (this.props.user) this.syncUser();
   }
   syncUser = () => {
-    gtm.updateUser(this.props.user);
+    gtm.updateUser(this.props.user.id);
```

Comment:
This was broken?

Reply:
No i just changed it to pass only the id, it’s easier to type instead of typing the entire user.
I guess with the generated types I could revert it back now

Reply:
Ah okay. I don‘t mind either way.